### DISCUSSION TITLE
Is there a guide on how to update the non-docker version?

### DISCUSSION THREAD

**Discussion by jasondad36** [tobychui/zoraxy#234] [General]
Hello,
I have this installed from source on a ubuntu 20.04 vm. I am not super familiar with GO, so i followed the instructions and it worked fine. But i would like to update it to the most recent version. Is there a guide on how to do that when not using docker?

**Comment by tobychui** [comment]
Just download the new version binary file and overwrite the old one. (Use `systemctl stop zoraxy` and `systemctl start zoraxy` if needed, notice that some versions might contains breaking changes. Remember to backup your files and configs before update.)

**Reply by yumilus** [reply]
Hello.
Would you implement the ability to perform updates via GUI from the admin screen?
I am tired of doing Git in the CLI.

Best regards.

**Reply by tobychui** [reply]
@yumilus PRs are always welcomed :)
